gamefixes-steam/409720.py

```diff
@@ -4,12 +4,7 @@
 
 
 def main() -> None:
-    """Disable ESYNC, disable intro's"""
-    # After loading the game, or a save file, a key needs to be pressed
-    # to continue. That screen does not respond to keyboard or mouse,
-    # so there is no way to continue. -nointro disables that screen
-    # (but also the intro's at the start of the game).
-    util.append_argument('-nointro')
```

Comment:
I'm deciding that we should keep the fsync and esync related fixes for those titles while removing the disable intro argument until Kisak is able to substantiate their statement. [I've already submitted a comment to them](https://github.com/ValveSoftware/Proton/issues/482#issuecomment-988954117).